lms/static/js/dashboard/views/dashboard_course_list_factory.js

```diff
@@ -0,0 +1,10 @@
+;(function (define) {
+    'use strict';
+
+    define(['js/dashboard/views/course_list_tabbed_view'],
+        function (CourseListTabbedView) {
+            return function (currentCourses, archivedCourses, courseSettings) {
+                new CourseListTabbedView(currentCourses, archivedCourses, courseSettings).render();
```

Comment:
Instead of calling render on the new view you could update so that at the end of the View's initialize function it calls this.render(). Is there a use case for initializing the View and not rendering straight away?